### PR TITLE
Add ArenaQuickJoin button

### PR DESCRIPTION
**Pull request by iam3yal**
This PR adds the ArenaQuickJoin button to PvPAssistant and fixes the following issues:

1. Rename Pv**p**Assistant.toc to Pv**P**Assistant.toc because some file systems are case sensitive.
2. Add Libs/ to .gitignore to smooth out the development workflow.
3. Add .pkgmeta to create a functional release build. Tested with the packager.

### CODE REVIEW

File: .pkgmeta [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package-as: PvPAssistant
```
Comment (derfloh205) [inline .pkgmeta:1]: this file is especially for the bigwigs packager? Or does anything else "look" at it?

File: Classes/MatchHistory.lua [hunk 2]
```diff
@@ -1,255 +1,255 @@
...
-    matchHistory.duration = serializedData.duration
-    matchHistory.isRated = serializedData.isRated
-    matchHistory.pvpMode = serializedData.pvpMode
-    matchHistory.win = serializedData.win
-    matchHistory.season = serializedData.season
-    matchHistory.player = serializedData.player
-    matchHistory.pvpMatchState = serializedData.pvpMatchState
-    return matchHistory
-end
+---@class PvPAssistant
```
Comment (derfloh205) [inline Classes/MatchHistory.lua:1]: Could you check your auto format config? Tabs should be 4 spaces (I think) Ill check tmr to be sure

File: .gitignore [hunk 3]
```diff
@@ -1 +1,5 @@
-.vscode
\ No newline at end of file
+.vscode
+.release/
+Libs/
+!Libs/classic.lua
```
[the remark this answers is not in the document]
Comment (iam3yal) [reply .gitignore:4]: @derfloh205 Yes.

File: .pkgmeta (hunk 1 above)
Comment (iam3yal) [reply .pkgmeta:1]: This is only for the packager.

File: Classes/MatchHistory.lua (hunk 2 above)
Comment (iam3yal) [reply Classes/MatchHistory.lua:1]: I'm using 4 spaces. 🤔

**Pull request merged**